app/templates/transcripts.html

```diff
@@ -0,0 +1,39 @@
+<div class = 'container'>

```

Comment:
Looks like there is a lot of hard-coded stuff here.

Reply:
We decided to get rid of this file and will branch off this PR to work on the front end of this issue. Our new html file will inherit from base.html